wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,168 @@
 
-#### Usage Examples
+```
+13:03 pippi: !faq peppy
+13:03 BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+```
+
+<!-- used an an online OCR to get the text here. May delete this later -->
 
 ```
-pippi: !faq peppy
-BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+13:05 pippi: !faq list
+13:05 BanchoBot: Commands are: academy, achievements, add-skin, advertising, answer, appeal, approved, arabic, ask, aspire, attention, avatar-change 
+13:05 BanchoBot: bahasa, balkan, banchobot, beatmapping, beginner, bitrate, bn, bored, bug, bulgarian, ca-sa, cantonese 
+13:05 BanchoBot: capping, caps, cat, change-modes, change-name, changelog, chars, chart, chinese, chinese-input, chinese-pinyin, color 
+13:05 BanchoBot: colour, command-abuse, comment, comment-color, completion, conferencemode, conmigo, countfaq, country-maps, country-ranking, critic, ctb 
+13:05 BanchoBot: cutting-edge, cwc, czechoslovak, danish, dasdass, decency, disable-this, distance-snap, dont-ask-peppy, douche, download, drugs 
+13:05 BanchoBot: dutch, ebay, editor-shortcuts, english, english-help, evade, fallback, faq, feature, filipino, fitter, finnish 
+13:05 BanchoBot: flow, foul-play, french, friend, friends-list, fullscreen, german, glossary, gmt, grammar, greek, guide 
+13:05 BanchoBot: hacked, hard, hebrew, help, hitsound-library, hl, how-to-bn, how-to-get-modded, how-to-gmt, how-to-play, how-to-rank, hungarian
+13:05 BanchoBot: hyperdash, ignore, import, Indonesian, inheriting, irc, Italian, japanese, keys, kiai, korean, kudosu 
+13:05 BanchoBot: lag, language, language-specific, level, tines, link, lobby, localisation, long-link, long-text, toyed, mac 
+13:05 BanchoBot: malaysian, map-this, mapping, mat, mentoring, modhetp, modqueue, modreqs, mptimeroo, multi invite, multiaccounting, multiplayer 
+13:05 BanchoBot: mwc, nat, nightcore, noel, norwegian, np, nsfw, old-maps, osu, osu!droid, osugablet, osu!talk 
+13:05 BanchoBot: osucore, osumania, osustream, osz, OSZ2, owc, peppy, percolate, perfect, ping, pippi, piracy 
+13:05 BanchoBot: play-shortcuts, pm, polish, portuguese, pp, preview, problem, qat, qualified, ranked, ranking-criteria, redname 
+13:05 BanchoBot: report, report-abuse, request, respect, roll-abuse, romaji, romanian, room, rp, rules, russian, sauce 
+13:05 BanchoBot: sb, scorevz, scoring, seitseid, silence, skandinavian, skinning, sleep, songs-not-appearing, spaces, spam, spanish 
+13:05 BanchoBot: spectate, spoiler, status, store, submit, suck, supporter, supporter-please, swedish, tablet, tag, taiko 
+13:05 BanchoBot: taiko-mapping, taiko-maps, thai, topic, training, tsya, turkish, tutorial, twc, ukrainian, unicode, upload-cap 
+13:05 BanchoBot: utfa, videogames, vietnamese, welcome, when, wiki, zallius 
```

Comment:
this is excessive and does not really contribute to the article: the entries may be added or removed, not to mention it's really a wall of text